Source/VSProps/Base.props

```diff
@@ -111,7 +112,7 @@
       4324  Padding was added at the end of a structure because you specified a __declspec(align) value.
       4714  function 'function' marked as __forceinline not inlined
       -->
-      <DisableSpecificWarnings>4201;4127;4100;4244;4121;4324;4714;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <DisableSpecificWarnings>4200;4201;4127;4100;4244;4121;4324;4714;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
Since this was only needed for a few files (which included libusb.h), I had opted to disable the warning only for those files. If we're going to disable it globally, the other disables can be removed now.